app/scripts/modules/core/pipeline/config/stages/deploy/deployStage.js

```diff
@@ -91,7 +91,7 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.deployStage', [
               return $scope.application;
             },
             serverGroupCommand: function () {
-              return serverGroupCommandBuilder.buildNewServerGroupCommandForPipeline(selectedProvider, $scope.stage, $scope.$parent.pipeline.stages);
+              return serverGroupCommandBuilder.buildNewServerGroupCommandForPipeline(selectedProvider, $scope.stage, $scope.$parent.pipeline);
```

Comment:
I changed this to `pipeline` to allow finding images from triggers in the future.